apex_launchtest/apex_launchtest/apex_launchtest_main.py

```diff
@@ -29,7 +30,10 @@
 def _load_python_file_as_module(python_file_path):
     """Load a given Python launch file (by path) as a Python module."""
     # Taken from apex_core to not introduce a weird dependency thing
-    loader = SourceFileLoader('python_launch_file', python_file_path)
+    loader = SourceFileLoader(
+        os.path.basename(python_file_path),
```

Comment:
This will give the tests a better name in the resulting XML.  The `TestRun` class in `loader.py` uses the module name to construct the name of the test run.  This information eventually makes it into the junit XML file and makes it a little easier to see where tests came from, and what parameters they ran with.